department-of-veterans-affairs/caseflow
# J hoang/appeals 44940

Resolves [APPEALS-44940](https://jira.devops.va.gov/browse/APPEALS-44940)

# Description
Updated change history query in business_line.rb to account for the Remand decision review type

## Acceptance Criteria

## Testing Plan
1. Create some remands from rails console `FactoryBot.create_list(:remand_vha_task, 3)`
2. Also create some SupplementalClaims and HigherLevelReviews (to help discern differences in claim type) if you don't have any
`FactoryBot.create_list(:supplemental_claim_vha_task, 3)`
`FactoryBot.create_list(:higher_level_review_vha_task, 3)`
3. Login as a VHA admin and navigate to the generate report page by visiting the url `/decision_reviews/vha/report`
4. Click on any report type
5. Press the Add Condition button
6. Set the condition variable to Decision Review Type
7. Choose Remands as an option
8. Click Generate task report
9. Verify only Remands data show in CSV.
10. Now go back and add another Decision Review Type condition along with Remands (add a Supplemental Claim or Higher-Level Review)
11. Click Generate task report
12. Verify Remands and the other decision review type data (you picked in step 9)shows in the CSV.
13. Lastly, remove all conditions and generate task report.
14. Verify all types of claim types show in the CSV including the newly added Remands.
15. You may also add all three claim types to your condition to verify the correct claim types show in the CSV but this is the same as no decision review type condition


## Storybook Story
*For Frontend (Presentation) Components*

# Backend
## Database Changes
*Only for Schema Changes*


## Integrations: Adding endpoints for external APIs

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
